validator/src/main/resources/expected-data-template/dotnet/ec2/asg/aws-sdk-call-log.mustache

```diff
@@ -0,0 +1,20 @@
+[{
+  "EC2.InstanceId": "^{{instanceId}}$",
+  "Environment": "^ec2:{{platformInfo}}$",
+  "PlatformType": "^AWS::EC2$",
+  "Operation": "GET aws-sdk-call",
+  "Version": "^1$",
+  "Telemetry.Source": "^LocalRootSpan$",
+  "Host": "^{{privateDnsName}}$"
+},
```

Comment:
Test is for ASG and isn't checking `EC2.AutoScalingGroupName`?

Reply:
Let's double check all the telemetry validations before the next round of review to ensure that we are checking the informations we are interested in. If this is copied from other tests and it's missing there as well, raise that up and we should fix it immediately.

Reply:
Regarding this, I believe it don't produce, same as python? Instead, it reflects in traces - metadata - EC2.AutoScalingGroup